deps/build.jl

```diff
@@ -21,7 +21,7 @@ if VERSION == v"0.7"
         Linux(:x86_64, :glibc) => ("$bin_prefix/libcxxwrap-julia-0.7.v0.4.0.x86_64-linux-gnu.tar.gz", "7dafd9db3dd898151ce54bf52ff5b6d7dc417a70fc88ac2594d73c53f284c461"),
         Windows(:x86_64) => ("$bin_prefix/libcxxwrap-julia-0.7.v0.4.0.x86_64-w64-mingw32.tar.gz", "e634a847567de9b3335281dad9d452f1dd6a1b4dc885740ea79463bde16de9e1"),
     )
-elseif VERSION == v"1.0"
+elseif VERSION == v"1.0" || VERSION == v"1.0.1"
```

Comment:
I'd suggest trying with
```julia
elseif VERSION >= v"1.0" && VERSION < v"1.1"
```

Reply:
That makes more sense. Especially if you don't think the minor version changes should interfere with CxxWrap.